otherlibs/unix/unix.mli

```diff
@@ -675,11 +675,39 @@ val close_process_full :
 (** {6 Symbolic links} *)
 
 
-val symlink : string -> string -> unit
-(** [symlink source dest] creates the file [dest] as a symbolic link
-   to the file [source].
-
-   On Windows: not implemented. *)
+val symlink : ?is_dir:bool -> string -> string -> unit
+(** [symlink ?is_dir source dest] creates the file [dest] as a symbolic link
+   to the file [source]. On Windows, [~is_dir] indicates if the symbolic link
+   points to a directory or a file; if omitted, [symlink] examines [source]
+   using [stat] and picks appropriately, if [source] does not exist then [false]
+   is assumed. On Unix, [~is_dir] ignored.
```

Comment:
Thanks - two further good changes. I agree it'd be good to recommend that new code should specify the option parameter and also that `~to_dir` is a good final iteration for the name of it!